test/tabs_test.js

```diff
@@ -93,6 +93,7 @@ describe('Tabs', function() {
 
   describe('selecting new tab (2)', function() {
     before(function(done) {
+      const browser = new Browser();
```

Comment:
Is closing all the tabs using **browser.tabs.closeAll();** supposed to remove the event listeners too? If so then yes there's a bug somewhere in the code. Because from what I noticed the event listener from the previous test was not removed, thus triggering **done()** multiple times and failing the test!